pages/index.vue

```diff
@@ -1,9 +1,53 @@
 <template>
-  <NuxtLink :to="`/usecase_0/panel_1`">
-    Example use case
-  </NuxtLink>
+  <div class="flex flex-col w-screen h-screen bg-gray-200 p-2">
+    <!-- Banner -->
+    <div class="banner">
+      <Banner />
+    </div>
+
+    <div class="flex overflow-auto">
+      <!-- usecases -->
+      <div class="m-2 bg-white rounded">
+        <!-- category 1 -->
+        <h3 class="text-2xl">
+          Category 1
+        </h3>
+        <div class="flex text-left">
+          <UseCaseCard
+            v-for="usecase in usecases.items"
+            :key="usecase.id"
+            :card="usecase"
+          />
+        </div>
+        <!-- category 2 -->
+        <h3 class="text-2xl">
+          Category 2
+        </h3>
+      </div>
```

Comment:
Maybe we can skip the catogories for now and just add them later on once we really need them. But good to see that it's quite straightforward

Reply:
Sounds good! I remove the other categories.